ocaml/testsuite/tests/lib-runtime-events/test_caml.ml

```diff
@@ -1,12 +1,25 @@
 (* TEST
+ {
+   runtime4;
+   skip;
+ }{
```

Comment:
Is this block still necessary if we
put `runtime5` in the other one?

Reply:
Maybe not, but it was there already, so I think this is ok.  We'll come back to the runtime_events tests in due course anyway.

Reply:
(that will indeed equally apply to the commented-out version below)